src/main/java/frc/robot/Constants.java

```diff
@@ -176,9 +173,13 @@ public AimModes previous() {
   public static final int middleSensorActivation = 450;
   public static final int backSensorActivation = 600;
 
-
   // spotless:off
   // Controller Constants {
+
+    // Controller modes
+    public static final boolean TestController = true;
```

Comment:
The naming of the variable is vague. Does true mean we are testing therefore using only 1 controller? oneController seemed a bit more clear. 

At least add comments to describe what true and false mean, but also consider changing the name to make it more clear.

Reply:
Now that we have a way to detect how many controllers we have there is no need for those containts.